org.eclipse.winery.bpmn4tosca.converter.tobpel/src/main/java/org/eclipse/winery/bpmn2bpel/parser/Bpmn4JsonParser.java

```diff
@@ -79,22 +88,24 @@ public ManagementFlow parse(URI jsonFileUrl) throws ParseException {
 				 * management tasks expected which are transformed to tasks in
 				 * our management model
 				 */
-				Task task = createTaskFromJson(jsonNode);
+				Node node = createTaskFromJson(jsonNode);
```

Comment:
This feels not correct. Either rename the method "createTaskFromJson". Also please adapt the comment on line 93.